pogobuf/pogobuf.client.js

```diff
@@ -804,7 +826,11 @@ function Client() {
 
     this.maxTries = 5;
     this.mapObjectsThrottlingEnabled = true;
-    this.mapObjectsMinDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMaxDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMinDistance = DEFAULT_MAP_OBJECTS_DISTANCE;
+    this.interactionThrottlingEnabled = true;
```

Comment:
Correct me if I'm wrong, but I think the API returns a failure if you are too far away anyway, so this would be preventing an unnecessary API call.